aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -20,14 +24,17 @@ module aptos_std::ristretto255_bulletproofs {
     // Error codes
     //
 
-    /// There was an error deserializing the range proof.
-    const E_DESERIALIZE_RANGE_PROOF: u64 = 1;
-
     /// The committed value given to the prover is too large.
-    const E_VALUE_OUTSIDE_RANGE: u64 = 2;
+    const E_VALUE_OUTSIDE_RANGE: u64 = 1;
```

Comment:
I don't think you should be changing these codes. I think it could/will break backwards compatibility.

Reply:
Done